example_agents/rl_agents/README.txt

```diff
@@ -0,0 +1,17 @@
+# Installing tensorflow on Apple Silicon
```

Comment:
Maybe add a brief description here on how to run the agents in this directory.

Reply:
done.